src/VisualStudio/CSharp/Test/Interactive/InteractiveWindowTestHost.cs

```diff
@@ -25,9 +25,9 @@ public sealed class InteractiveWindowTestHost : IDisposable
                     typeof(TestWaitIndicator).Assembly,
                     typeof(TestInteractiveEvaluator).Assembly,
                     typeof(IInteractiveWindow).Assembly
-                }
-                .Concat(TestExportProvider.GetCSharpAndVisualBasicAssemblies())
-                .Concat(MinimalTestExportProvider.GetEditorAssemblies())));
+                })
+                .WithParts(TestExportProvider.GetCSharpAndVisualBasicAssemblyCatalog())
+                .WithParts(MinimalTestExportProvider.GetEditorAssemblyCatalog()));
```

Comment:
why this change?

Reply:
This is because of change from IEnumerable<Assembly> to ComposableCatalog in TestExportProvider. We can consider it separately. If I'm not mistaken, @sharwell suggested to include this change.

---
In reply to: [232508050](https://github.com/dotnet/roslyn/pull/29016#discussion_r232508050) [](ancestors = 232508050)

Reply:
@sharwell, do you prefer to keep them to finally merge them? or exclude them from the PR?

---
In reply to: [233079720](https://github.com/dotnet/roslyn/pull/29016#discussion_r233079720) [](ancestors = 233079720)

Reply:
@jasonmalinowski, this change was added by @sharwell suggestions

---
In reply to: [234826635](https://github.com/dotnet/roslyn/pull/29016#discussion_r234826635) [](ancestors = 234826635)